our-work/software/materials-db.yml

```diff
@@ -1,7 +1,7 @@
 title: 2D Materials Database
 description: A normalized database for Bob Hurt's lab to track and publish data on toxicity of 2D materials.
 groups:
-  - hurt
+  - hurt-lab
```

Comment:
does the file need to be renamed now? The instructions say to use the identifier used in the file name.

Reply:
😅 none of the files-names have the `lab`,  but all of the group names have `lab`. I'm impartial. The instructions are wrong in any case because they say we should match the `slug`, but I got rid of that field in the collaborations yaml. This is related to a "nice to have" we've talked about of being able to link collaborations and software. I think it would be simpler to link to a field that comes from the `yaml` instead of the file name, but then we need to put in that field then. Son either bring back `slug` or have `id` or something?

Reply:
ah, I had assumed slug was file name because of how nuxt-content turns the directory into an api

Reply:
I think `id` is more human interpretable than `slug`

Reply:
okay, I'll add and id field in `collaborations` and adjust the instructions